src/data/legacy/processes.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "Processes"
+description: "View and remove processes."
+---
+
+# List
+
+Get all existing processes.
+
+<Endpoint method="GET" path="/api/processes" />
+
+## Example
+
+<Request method="GET" path="/api/processes" />
+
+## Response
+
+<Response status="200 OK">
+
+```json
+[
+	{
+		"created_at": "2018-05-02T23:04:15.376000Z",
+		"progress": 1,
+		"step": "create_history",
+		"step_count": 0,
+		"type": "import_reference",
+		"id": "pqbed16v"
+	},
+	{
+		"created_at": "2018-05-02T23:11:38.494000Z",
+		"progress": 1,
+		"step": "create_history",
+		"step_count": 0,
+		"type": "import_reference",
+		"id": "d9tn5khk"
+	}
+]
```

Comment:
The indentation looks excessive here. Make sure it matches the other sections. Check other JSON in this PR as well.